src/components/toolbar.tsx

```diff
@@ -116,8 +116,8 @@ export class ToolbarComponent extends BaseComponent<IProps, {}> {
 
   private handleDelete() {
     const { document } = this.props;
-    const { ui: { selectedTileId } } = this.stores;
-    if (selectedTileId) {
+    const { ui: { selectedTileIds } } = this.stores;
+    selectedTileIds.forEach(selectedTileId => {
```

Comment:
I am concerned about whether this will be intuitive to users. Multiple selected tiles, each potentially having selected content within seems like it could lead to user confusion over what's about to be deleted. It seems to me that once a user has gone to the trouble of holding down a modifier key to intentionally selected multiple tiles that they're probably thinking at the tile level rather than the intra-tile level, and so we may want to deselect internal content at that point. But since we don't have a ready mechanism for doing that right now, maybe we start here and see how it goes.